src/server/index.js

```diff
@@ -60,7 +64,25 @@ async function createServer() {
     await server.register(secureContext)
   }
 
-  await server.register([router, sqsListener])
+  await server.register([router])
+
+  await server.register({
+    plugin: sqsListener,
```

Comment:
is this working? should this not be `plugin: sqsListener.plugin`?

Reply:
yep